src/test/java/scenarios/webHomeworkTest.java

```diff
@@ -0,0 +1,30 @@
+package scenarios;
+
+import java.net.MalformedURLException;
+import org.openqa.selenium.Keys;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pageObjects.WebPages.GooglePage;
+import setup.BaseTest;
+
+public class webHomeworkTest extends BaseTest {
+
+    //test will be started with other "web" tests
+    @Test(groups = {"web"}, description = "Check EPAM word search results")
+    public void WebTest() throws InterruptedException {
+
+        //load google search page
+        getDriver().get("https://www.google.com");
+
+        String searchText = "EPAM";
```

Comment:
It would be good to keep constants out of test methods, as constants or in a property file

Reply:
Fixed